php/tests/WellKnownTest.php

```diff
@@ -416,4 +416,44 @@ public function enumNameValueConversionDataProvider()
             ['\Google\Protobuf\Syntax'],
         ];
     }
+
+    /**
+     * @dataProvider optionalFieldsDataProvider
+     */
+    public function testReflectionProperty($property, $default)
+    {
+        $testMessage = new TestMessage();
+
+        $functionName = implode('', array_map('ucwords', explode('_', $property)));
+
+        $reflectionProperty = new \ReflectionProperty($testMessage, $property);
+
+        self::assertFalse(call_user_func([$testMessage, 'has'.$functionName]));
+        self::assertEquals($default, call_user_func([$testMessage, 'get'.$functionName]));
+        self::assertNull($reflectionProperty->getValue($testMessage));
```

Comment:
These tests fail when testing without the protobuf extension installed... probably because the change made above is only when the `protobuf.so` extension is enabled. 

<details><summary>Failing tests</summary>

```
$ vendor/bin/phpunit tests --filter testReflectionProperty
PHPUnit 8.5.26 #StandWithUkraine

FFFFFFFFFFFFFFFF..                                                18 / 18 (100%)

Time: 46 ms, Memory: 8.00 MB

There were 16 failures:

1) WellKnownTest::testReflectionProperty with data set #0 ('true_optional_int32', 0)
Failed asserting that 0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

2) WellKnownTest::testReflectionProperty with data set #1 ('true_optional_int64', 0)
Failed asserting that 0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

3) WellKnownTest::testReflectionProperty with data set #2 ('true_optional_uint32', 0)
Failed asserting that 0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

4) WellKnownTest::testReflectionProperty with data set #3 ('true_optional_uint64', 0)
Failed asserting that 0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

5) WellKnownTest::testReflectionProperty with data set #4 ('true_optional_sint32', 0)
Failed asserting that 0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

6) WellKnownTest::testReflectionProperty with data set #5 ('true_optional_sint64', 0)
Failed asserting that 0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

7) WellKnownTest::testReflectionProperty with data set #6 ('true_optional_fixed32', 0)
Failed asserting that 0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

8) WellKnownTest::testReflectionProperty with data set #7 ('true_optional_fixed64', 0)
Failed asserting that 0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

9) WellKnownTest::testReflectionProperty with data set #8 ('true_optional_sfixed32', 0)
Failed asserting that 0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

10) WellKnownTest::testReflectionProperty with data set #9 ('true_optional_sfixed64', 0)
Failed asserting that 0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

11) WellKnownTest::testReflectionProperty with data set #10 ('true_optional_float', 0.0)
Failed asserting that 0.0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

12) WellKnownTest::testReflectionProperty with data set #11 ('true_optional_double', 0.0)
Failed asserting that 0.0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

13) WellKnownTest::testReflectionProperty with data set #12 ('true_optional_bool', false)
Failed asserting that false is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

14) WellKnownTest::testReflectionProperty with data set #13 ('true_optional_string', '')
Failed asserting that '' is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

15) WellKnownTest::testReflectionProperty with data set #14 ('true_optional_bytes', '')
Failed asserting that '' is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

16) WellKnownTest::testReflectionProperty with data set #15 ('true_optional_enum', null)
Failed asserting that 0 is null.

/Users/betterbrent/Code/protobuf/php/tests/WellKnownTest.php:433

FAILURES!
Tests: 18, Assertions: 54, Failures: 16.
```
</details>